tests/chimera-host/host/returnZero/CMakeLists.txt

```diff
@@ -30,4 +30,20 @@ add_chimera_test(
 # WIESEP: Link the host to the test executable
 target_link_libraries(${TEST_NAME} PUBLIC ${TEST_NAME}_host)
 
+################################################################################
+# Debugging Options                                                             #
+################################################################################
+
+target_compile_options(${TEST_NAME}_host
+  PUBLIC
+  -ffunction-sections
+  -fdata-sections
+)
+
+target_link_options(${TEST_NAME}
+  PUBLIC
+  -Wl,--print-memory-usage
```

Comment:
`--print-memory-usage` is GCC only, irc.

Reply:
Since release 17.0.1 it is also would also be supported by the LLD linker. However, with the current flow, we use the LD linker for the LLVM and GCC build flow. Hence, I do not see an issue, and it is tested with the current setup, using `toolchain_llvm.cmake`.

References:
- https://releases.llvm.org/17.0.1/tools/lld/docs/ReleaseNotes.html